app/lib/repository/client.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:effektio_flutter_sdk/effektio_flutter_sdk.dart';
+
+Future<Client> makeClient() async {
+  final sdk = await EffektioSdk.instance;
+  Client client = await sdk.currentClient;
+  return client;
+}
+
+Future<Client> login(String username, String password) async {
+  final sdk = await EffektioSdk.instance;
+  Client client = await sdk.login(username, password);
+  return client;
+}
```

Comment:
I'd prefer if we need helper methods, that we directly put them into the SDK rather than another intermediary. the file `package:effektio_flutter_sdk/effektio_flutter_sdk.dart` is exactly for that and isn't overwritten.

Reply:
It is defined in seperate file for a purpose that if in future more methods get populated, they can be defined in repository. Also to seperate flutter specific functions that are used in UI, it will be more understandable.